lesson6/src/test/java/com/andersenlab/hashSet/CustomHashSetTest.java

```diff
@@ -0,0 +1,61 @@
+package com.andersenlab.hashSet;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Iterator;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Try avoid generic imports with asterisk sign for better readability and maintainability